src/inabox/inabox-resources.js

```diff
@@ -183,6 +212,41 @@ export class InaboxResources {
     this.passObservable_.fire();
     this.firstPassDone_.resolve();
   }
+
+  /**
+   * @return {?IntersectionObserver}
+   * @private
+   */
+  initIntersectionObserver_() {
+    if (this.intersectionObserver_) {
+      return this.intersectionObserver_;
+    }
+    const {IntersectionObserver} = this.win;
+    if (!IntersectionObserver) {
+      return null;
+    }
+    return (this.intersectionObserver_ = new IntersectionObserver(
+      entries => entries.forEach(triggerViewportCallbackFromIntersection),
+      {threshold: 0.5}
```

Comment:
0.5 seems very strange. We usually do `threshold: 0` for this. No?

Reply:
Used for symmetry with how video autoplay works. No strong opinion either way, removed to use default `0`.

Reply:
To clarify, 0 is only b/c the main doc Resources does 0 as well.